guides/development-workflow.md

```diff
@@ -0,0 +1,284 @@
+  * give it an [appropriate name and comment]()  
+3. Link the Trello card to the new pull request, linking directly or using the [power up](http://help.trello.com/article/1021-integrating-trello-with-zendesk)
+3. move the Trello card into the 'Awaiting column'
+
+### Reviewing another developer's pull request
+
+1. move the Trello card into the 'Under code review' column
+2. carry out a [code review]()
+  * if it's not ready:
+    - leave feedback
+    - mention the author so they're aware
+  * if it is ready:
+    - press the merge button
+    - delete the leftover branch
+    - move the Trello card into 'Deployed to Staging'
+
+---
+## The lifecycle of a ticket
+
+Tickets can fall into two types of work flow depending on their severity.
+
+If there an emergency that needs to be on fixed production as soon as possible? These are rare, but if so reach for a `hotfix` else a normal `fix`, which behaves just like a [story]().
+
+### Issue a hotfix
+  1. create a new card in Trello and move it to 'In progress'
+  2. create a new branch off of master
+  ```
+  git fetch -a
+  git checkout master
+  git pull
+  git branch <branch-name>
+  git checkout <branch-name>
+  ```
+  3. commit the fix
+  4. make a pull request into master
+  5. move the Trello card into 'Awaiting review'
+
+### Review and deploy a hotfix
+  1. review the code
+  2. check the fix is tested locally if possible
+  3. merge the pull request
+  4. manually merge the pull request back into develop
+  ```
+  git checkout develop
+  git pull
+  git merge origin/master
+  git push
+  ```
+  5. move the card into 'Deployed to production'
+
+---
+
+## Branching
+
+#### How to make a branch
+
+Create the branch off of develop:
+```
+git fetch
+git checkout develop
+git branch <branch-name>
+git checkout <branch-name>
+```
+
+**Naming the branch**
+
+Having a convention allows for:
+
+* all branches connected to a story or ticket are easily connected now and in the future
+* others can easily identify the intent of the contents
+
+**Convention:**
+
+* Feature - A branch that adds a new feature, as defined by the specified story
+* Fix - A branch that corrects a problem in a feature already merged into develop
+* Chore - Chore branches are used for routine tasks or tickets which are not emergencies
+* Hotfix - A branch that adds an urgent fix to a problem that affects production. These branches are based on master and do not go via develop, so must be also be merged into develop when deployed
+
+```
+feature/{STORY ID}-{CONCISE TITLE}
+fix/{STORY/TICKET ID}-{CONCISE TITLE}
+chore/{STORY ID}-{CONCISE TITLE}
+hotfix/{TICKET ID}-{CONCISE TITLE}
+```
+
+**Examples:**
+```
+feature/523797477-add-logging-to-registration
+fix/423797477-logging-happens-in-both-environments
+chore/1934729239-reduce-caching-for-contact-details
+hotfix/253947623-remove-breaking-change-to-repair-creation
+```
+
+#### Update your branch with the latest changes on develop (rebasing)
```

Comment:
From my understanding it was not the process of rebasing that was the issue just the confidence and understanding of when exactly to execute them. It was only one of the points behind the to switch to a consistent development process, which was primarily discussed at the most recent retreat.